src/main/java/com/softserve/paymentservice/dto/InvoiceDto.java

```diff
@@ -0,0 +1,19 @@
+package com.softserve.paymentservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class InvoiceDto {
+    private UUID userUUID;
+    private int amount;
+    private boolean payed;
+    private String currency;
+    private Date date;
```

Comment:
Or LocalDateTime

Reply:
1. LocalDateTime is applicable for date that won't be usually compared. Or when timezone doesn't really matters.
2. ZonedDateTime is needed to show user's time not only with offset but with a timezone too (e.g. Europe/Kiev). For example, It's used in messengers, so that one user can see a time of another user. But for this case, an OffsetDateTime can be used too. OffsetDateTime has offset e.g. UTC +2, but it doesn't handle daylight savings, so it's not fully appropriate for the time displayed to the user.
3. Instant is used for timestamps, values that usually will be compared, but won't be shown for the typical user. This field is the case for the Instant. Instants have neither offset nor timezone (though they are almost always equal to UTC time). Instant is just a moment in time and it'll be equal in any timezone. As we don't need neither timezone nor offset, Instant is the most appropriate here.